API.Tests/Services/ParseScannedFilesTests.cs

```diff
@@ -349,4 +357,101 @@ public async Task ProcessFiles_ShouldCallFolderActionOnce()
 
 
     #endregion
+
+    [Fact]
+    public async Task HasSeriesFolderNotChangedSinceLastScan_AllSeriesFoldersHaveChanges()
+    {
+        const string testcase = "Subfolders always scanning all series changes - Manga.json";
+        var infos = new Dictionary<string, ComicInfo>();
+        var library = await _scannerHelper.GenerateScannerData(testcase, infos);
+
+        var testDirectoryPath = Path.Join(
+            Path.Join(Directory.GetCurrentDirectory(), "../../../Services/Test Data/ScannerService/ScanTests"),
+            testcase.Replace(".json", string.Empty));
+        testDirectoryPath = Path.GetFullPath(testDirectoryPath);
```

Comment:
Instead of applying this here, instead apply it on the return of ScannerHelper.GenerateTestDirectory() so that you can remove this library.Folders hack and all other tests will work for you.